src/test/perl/rbe_value_format.t

```diff
@@ -140,35 +140,64 @@ Readonly my @TEST_ARRAY => ('confFile', 'logFile', 'unused', 'logKeep', 'logFile
 Readonly my $FORMATTED_ARRAY => 'confFile logFile unused logKeep logFile';
 Readonly my $FORMATTED_ARRAY_SORTED => 'confFile logFile logFile logKeep unused';
 Readonly my $FORMATTED_ARRAY_UNIQUE => 'confFile logFile logKeep unused';
-my $rbe_fh = CAF::RuleBasedEditor->open($FILENAME, log => $obj);
+# When LINE_VALUE_OPT_SINGLE is set, formatAttributeValue() does nothihng and returns the array
+Readonly my $FORMATTED_ARRAY_SINGLE => 'ARRAY\(0x[a-f\d]+\)';
+$rbe_fh = CAF::RuleBasedEditor->open($FILENAME, log => $obj);
 ok(defined($rbe_fh), $FILENAME." was opened");
 $formatted_value = $rbe_fh->_formatAttributeValue(\@TEST_ARRAY,
-                                                  LINE_FORMAT_KEY_VAL,
+                                                  LINE_FORMAT_KW_VAL,
                                                   LINE_VALUE_ARRAY,
                                                   0,
                                                  );
 is($formatted_value, $FORMATTED_ARRAY, "Array values correctly formatted");
 $formatted_value = $rbe_fh->_formatAttributeValue(\@TEST_ARRAY,
-                                                  LINE_FORMAT_KEY_VAL,
+                                                  LINE_FORMAT_KW_VAL,
                                                   LINE_VALUE_ARRAY,
                                                   LINE_VALUE_OPT_SORTED,
                                                  );
 is($formatted_value, $FORMATTED_ARRAY_SORTED, "Array values (sorted) correctly formatted");
 $formatted_value = $rbe_fh->_formatAttributeValue(\@TEST_ARRAY,
-                                                  LINE_FORMAT_KEY_VAL,
+                                                  LINE_FORMAT_KW_VAL,
                                                   LINE_VALUE_ARRAY,
                                                   LINE_VALUE_OPT_UNIQUE,
                                                  );
 is($formatted_value, $FORMATTED_ARRAY_UNIQUE, "Array values (unique) correctly formatted");
+$formatted_value = $rbe_fh->_formatAttributeValue(\@TEST_ARRAY,
+                                                  LINE_FORMAT_KW_VAL,
+                                                  LINE_VALUE_ARRAY,
+                                                  LINE_VALUE_OPT_SINGLE,
+                                                 );
+like($formatted_value, qr/$FORMATTED_ARRAY_SINGLE/, "Array values (single) correctly formatted");
```

Comment:
the comment above `$FORMATTED_ARRAY_SINGLE` says otherwise. also the pattern itself looks like a stringified arrayref, instead of a newline joined list of strings.